### PR TITLE
add password policy on UI

### PR DESCRIPTION
**Pull request by achen2401**
Per https://www.pivotaltracker.com/story/show/179541021
Indicate on the UI the password policy

### CODE REVIEW

File: login/messages/messages_en.properties [hunk 1]
```diff
@@ -70,6 +70,7 @@ password=Password
 passwordConfirm=Confirm password
 passwordNew=New Password
 passwordNewConfirm=New Password confirmation
+passwordPolicy=<p>Password must be:</p><ul><li>Require at least one upper, lower, and numeric digit</li><li>minimum of 8 characters</li><li>not used recently</li></ul>
```
Comment (mcjustin) [inline login/messages/messages_en.properties]: @achen2401  I'd rephrase this to `<p>Password must:</p><ul><li>Include at least one upper case letter, one lower case letter, and one digit</li><li>Be at least 8 characters long</li><li>Have not been used recently</li></ul>`

Comment (achen2401) [reply login/messages/messages_en.properties]: thank you @mcjustin. fixed.

**Review by ivan-c — APPROVED** [review]
